src/bin/validator.rs

```diff
@@ -106,13 +105,10 @@ impl From<&CliOpts> for AppContext {
             file.parse().unwrap()
         } else {
             let n = config.bundler_public.as_ref().unwrap();
-            JsonWebKey::new(Key::RSA {
-                public: RsaPublic {
-                    e: PublicExponent,
-                    n: n.as_bytes().into(),
-                },
-                private: None,
-            })
+            let jwt_str = format!("{{\"kty\":\"RSA\",\"e\":\"AQAB\",\"n\":\"{}\"}}", n);
+            let jwk: JsonWebKey = jwt_str.parse().unwrap();
+            let p = serde_json::to_string(&jwk).unwrap();
+            p.parse().unwrap()
```

Comment:
Why to switch form building the object into parsing it from a string?

At least, rename to `jwt_str` to `jwk_str`.